platform-sdk/swirlds-common/src/main/java/module-info.java

```diff
@@ -160,6 +160,7 @@
             com.swirlds.platform.test,
             com.swirlds.platform.gui;
     exports com.swirlds.common.startup;
+    exports com.swirlds.common.threading.atomic;
```

Comment:
The rename also requires we remove the old name on line 47:
```
/runner/_work/hedera-services/hedera-services/platform-sdk/swirlds-common/src/main/java/module-info.java:47: error: package is empty or does not exist: com.swirlds.common.metrics.atomic
    exports com.swirlds.common.metrics.atomic
```

Reply:
THX, no idea why my local build ignored that. maybe gradle cache